Functions/Engines/Wine/script.js

```diff
@@ -177,6 +177,16 @@ Wine.prototype.run = function (executable, args, captureOutput) {
         args = [];
     }
 
+    var extensionFile = executable.split(".").pop();
+
+    if(extensionFile == "msi") {
+        return this.run("msiexec", ["/i", executable], captureOutput);
```

Comment:
True (because msiexec does not have the extension "msi"). Do you want me to merge?